<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerUp</class>
 <widget class="QWidget" name="registerUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogUp</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>enter account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="enterPasswordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>enter password</string>
   </property>
  </widget>
  <widget class="QLabel" name="enterPassword2Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>380</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
